template/.parcelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "@parcel/config-default",
+  "transformers": {
+    "*.{js,mjs,jsx,cjs,ts,tsx}": [
+      "@parcel/transformer-js",
+      "@parcel/transformer-react-refresh-wrap"
+    ]
+  }
+}
```

Comment:
This is pretty much a 1-for-1 copy from the [parcel documentation](https://parceljs.org/languages/javascript/#usage-with-other-tools) on how to handle needing a `babel.config.json` for other tools (like jest), but not for parcel.